doc/release-notes-1.12.rst

```diff
@@ -0,0 +1,80 @@
+:orphan:
+
+.. _zephyr_1.12:
+
+Zephyr Kernel 1.12.0 (DRAFT)
+############################
+
+We are pleased to announce the release of Zephyr kernel version 1.12.0.
+
+Major enhancements with this release include:
+
+* Asymmetric multiprocessing (AMP) on supported on multi-core platforms
```

Comment:
I would prefer if we only added the things that are already merged and closed per the roadmap, not everything to avoid confusion.